packages/o/openssl3/xmake.lua

```diff
@@ -0,0 +1,147 @@
+package("openssl3")
+
+    set_homepage("https://www.openssl.org/")
+    set_description("A robust, commercial-grade, and full-featured toolkit for TLS and SSL.")
+
+    add_urls("https://github.com/openssl/openssl/archive/refs/tags/openssl-$(version).zip")
+    add_versions("3.0.7", "fcb37203c6bf7376cfd3aeb0be057937b7611e998b6c0d664abde928c8af3eb7")
+    add_versions("3.0.6", "9b45be41df0d6e9cf9e340a64525177662f22808ac69aee6bfb29c511284dae4")
+    add_versions("3.0.5", "4313c91fb0412e6a600493eb7c59bd555c4ff2ea7caa247a98c8456ad6f9fc74")
+    add_versions("3.0.4", "5b690a5c00e639f3817e2ee15c23c36874a1f91fa8c3a83bda3276d3d6345b76")
+    add_versions("3.0.3", "9bc56fd035f980cf74605264b04d84497df657c4f7ca68bfa77512e745f6c1a6")
+    add_versions("3.0.2", "ce3cbb41411731852e52bf96c06f097405c81ebf60ba81e0b9ca05d41dc92681")
+    add_versions("3.0.1", "53d8121af1c33c62a05a5370e9ba40fcc237717b79a7d99009b0c00c79bd7d78")
+    add_versions("3.0.0", "1bdb33f131af75330de94475563c62d6908ac1c18586f7f4aa209b96b0bfc2f9")
+
+    on_fetch("fetch")
+
```

Comment:
install libssl.a -> /home/ruki/.xmake/packages/o/openssl3/3.0.7/8aaa08615b8740ada8faf167c0cbf065/lib64/libssl.a

add_linkdirs("lib64") for linux

Reply:
Thanks a lot 🙏 I was just checking if the libs' name was right.